### PR TITLE
DM-26731: fix a race condition in the CSC and lock the version of black in the conda file

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/MTDomeTrajectory/dome_trajectory.py [hunk 1]
```diff
@@ -288,15 +272,12 @@ async def move_dome_elevation(self, desired_dome_elevation):
 
             # Move the dome elevation axis and wait for the target event
             # and the first motion event (so motion has started).
-            self.log.debug("Move dome elevation")
-            if not flushed_motion_event:
-                self.dome_remote.evt_elMotion.flush()
             self.dome_remote.evt_elTarget.flush()
+            self.log.debug("Start a dome elevation move")
```
Comment (wvreeven) [inline python/lsst/ts/MTDomeTrajectory/dome_trajectory.py:276]: Would you be willing to consider making the log message "Start dome elevation motion" to be in line with line 264?

File: python/lsst/ts/MTDomeTrajectory/dome_trajectory.py [hunk 2]
```diff
@@ -338,17 +317,14 @@ async def move_dome_azimuth(self, desired_dome_azimuth):
 
             # Move the dome azimuth axis and wait for the target event
             # and the first motion event (so motion has started).
-            self.log.debug("Move dome azimuth.")
-            if not flushed_motion_event:
-                self.dome_remote.evt_azMotion.flush()
             self.dome_remote.evt_azTarget.flush()
+            self.log.debug("Start a dome azimuth move")
```
Comment (wvreeven) [inline python/lsst/ts/MTDomeTrajectory/dome_trajectory.py:321]: Would you be willing to consider making the log message "Start dome azimuth motion" to be in line with line 310?

**Review by wvreeven — APPROVED** [review]
Two small suggestions to possibly make logging more consistent and otherwise this looks good to me.

**Comment by r-owen** [comment]
Thank you. I made your two suggested changes.

**Pull request merged**